standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -60,6 +74,7 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class MetaStoreUtils {
```

Comment:
I see there is a MetaStoreUtils in hive-metastore project too. What is the relation between the two?

Reply:
I'm slowing moving pieces from one to the other.  There are a few things that may be left in metastore/MetaStoreUtils when the split is done, at which point it will probably make sense to rename it HiveMetaStoreUtils or something.